src/runtime/start/start-module.ts

```diff
@@ -40,7 +41,11 @@ export function createStartModuleContent(config: StartModuleConfig): string {
   content += stripIndent`
     // Run framework initialization side-effects
     // Also, import the app for later use
-    const app = require("nexus").default
+    const app = require("${
+      config.absoluteModuleImports
```

Comment:
```suggestion
      !config.willEmit
```

I think the idea here is about whether or not the content is going to be emitted. Everything is based on that question. If so, we can encode part of that into the name here.

Reply:
Well, then wouldn't `internalStage === 'dev'` be enough?

Reply:
I find `internalStage === 'dev'` confusing? What is dev? testing? dev mode? build on first pass to produce typegen? `willEmit` is just a straight forward local option, without dragging into scope a wider concept/discussion.